packages/theme/stories/introduction.stories.mdx

```diff
@@ -2,33 +2,29 @@ import { Meta } from "@storybook/addon-docs";
 
 <Meta title="Documentation/Styles and Theming/Introduction" />
 
-## UI Toolkit styling
+# Salt Theme
 
-You can use styling attributes from the UITK theme to style many elements of your application - native HTML used for structural and layout purposes, your own custom components, etc.
+## Our approach
 
-### Explaining our approach
+The Salt design system is built from Vanilla CSS, which enables us to provide a custom and consistent set of styles. There are several benefits to using the Vanilla CSS framework:
 
-For the Toolkit, our design system is built simply from vanilla CSS. There are a number of factors for this decision, including the following:
+- Salt has no third-party style dependencies, which means we are free to take our own approach to styling.
+- The Salt theme is built entirely using CSS custom properties.
+- Vanilla CSS provides performance improvements by removing the runtime overhead common in CSS-in-JS solutions.
+- Vanilla CSS allows the Salt theme to be published and consumed independent of the component library–It can be used in contexts where Salt components are not able to be used, e.g. static code deployment within a CMS.
```

Comment:
Our style guide recommends against using 'e.g.'. Suggest either the obvious 'for example' or '...such as environments with static code deployed within a CMS.'

Reply:
I've made the change, thanks.